media_driver/linux/common/ddi/media_libva.h

```diff
@@ -117,6 +117,11 @@
 // Enable unlimited output buffer, delete this build option (remove multiple output buffer) when it is verified
 #define ENABLE_ENC_UNLIMITED_OUTPUT
 
+// Max timeout for vaSyncSuraface2 and vaSyncBuffer
+#define DDI_MAX_TIMEOUT       (~(0x8000000000000000))
+// Negative value for infinite timeout
+#define DDI_INFINITE_TIMEOUT  (-1)
```

Comment:
Same as in https://github.com/intel/media-driver/pull/943: 
propose to extend the comment that -1 is infinite for i915